src/main/antora/modules/casestudies/pages/dukes-bookstore/dukes-bookstore002.adoc

```diff
@@ -115,8 +115,7 @@ See xref:web:faces-custom/faces-custom.adoc#_creating_custom_component_classes[C
 * `listeners`: Contains the `AreaSelectedEvent` class, along with other listener classes.
 See xref:web:faces-custom/faces-custom.adoc#_handling_events_for_custom_components[Handling Events for Custom Components].
 
-* `model`: Contains the `ImageArea` class.
```

Comment:
Can you leave the line above?

Reply:
okayy

Reply:
you can check now. I have added the above line and I have also fixed the issue #80.